File: app/src/main/java/pl/nataliana/didipoo/Data/MyPooReportContract.java
package pl.nataliana.didipoo.Data;

import android.provider.BaseColumns;

public class MyPooReportContract {

    public static abstract class PooEntry implements BaseColumns {

        public static final String TABLE_NAME = "MyPooHistory";

        public static final String _ID = BaseColumns._ID;
        public static final String COLUMN_DATE = "date";
        public static final String COLUMN_HOUR = "hour";
        public static final String COLUMN_DESCRIPTION = "description";
    }
}

File: app/src/main/java/pl/nataliana/didipoo/data/MyPooReportContract.kt
package pl.nataliana.didipoo.data

import android.provider.BaseColumns

class MyPooReportContract {

    abstract class PooEntry : BaseColumns {
        companion object {

            const val TABLE_NAME = "MyPooHistory"

            const val _ID = BaseColumns._ID
            const val COLUMN_DATE = "date"
            const val COLUMN_HOUR = "hour"
            const val COLUMN_DESCRIPTION = "description"
        }
    }
}